cmake/HelperBoost.cmake

```diff
@@ -0,0 +1,15 @@
+
+# Add to BOOST_ROOT variable a custom path to 
+# ease installation of giotto-tda on Windows platform
+# The custom path will be at `C:\\local\`
+if(WIN32)
+    list(APPEND BOOST_ROOT "C:/local")
+    list(APPEND BOOST_ROOT "") # Add custom path to your boost installation
+endif()
+
+message(STATUS "BOOST_ROOT: ${BOOST_ROOT}")
+
+find_package(Boost 1.56 REQUIRED)
+
+message(STATUS "Boost_INCLUDE_DIR: ${Boost_INCLUDE_DIR}")
+message(STATUS "Boost_INCLUDE_DIRS: ${Boost_INCLUDE_DIRS}")
```

Comment:
Can you link this and the above line with the discussion which took place in #354 ?

Reply:
In PR #354 , [discussion](https://github.com/giotto-ai/giotto-tda/pull/354#discussion_r389500141) and [discussion](https://github.com/giotto-ai/giotto-tda/pull/354#discussion_r389500703) some points on `boost` version and variables were addressed.

At the moment we can keep `boost` minimal require version to `1.56`, and from official documentation we keep paths provided by variable `Boost_INCLUDE_DIR`.
But in case of problem, I added some printing possibilities, if you need to understand where the compiler tries to look for `boost` headers, you can install `giotto-tda` as follow:

```bash
pip install -e . -v
```

The you can look at the output for lines starting with : 
```bash
 - Boost_INCLUDE_DIR: <path>
 - Boost_INCLUDE_DIRS: <path>
```

But from my understanding of the documentation, we should keep `Boost_INCLUDE_DIR`.

BTW, if you installed different versions of boost when you try to install `giotto-tda`, clearing CMake cache entries (`rm -rf build/`).

Reply:
Thanks @MonkeyBreaker: and the above instructions are what you've placed in #306 ?

Reply:
Not the part of printing the variables, maybe @wreise can have a look and integrate it directly in the `boost` section ?

Reply:
Thanks guys, I'll approve this PR but let's not forget to do this.